mne_bids/mne_bids.py

```diff
@@ -156,6 +156,71 @@ def _events_tsv(events, raw, fname, trial_type, verbose):
     return fname
 
 
+def _participants_tsv(raw, subject_id, group, fname, verbose):
+    """Create a participants.tsv file and save it.
+
+    This will append any new participant data to the current list if it
+    exists. Otherwise a new file will be created with the provided information.
+
+    Parameters
+    ----------
+    raw : instance of Raw
+        The data as MNE-Python Raw object.
+    subject_id : str
+        The subject name in BIDS compatible format ('01', '02', etc.)
+    group : str
+        Name of group participant belongs to.
+    fname : str
+        Filename to save the participants.tsv to.
+    verbose : bool
+        Set verbose output to true or false.
+
+    """
+    subject_info = raw.info['subject_info']
+    if subject_info is not None:
+        genders = {0: 'U', 1: 'M', 2: 'F'}
+        sex = genders[subject_info.get('sex', 0)]
+        subject_id = 'sub-' + subject_id
+
+        # determine the age of the participant
+        age = subject_info.get('birthday', None)
+        meas_date = raw.info.get('meas_date', None)
+        if meas_date is not None and age is not None:
+            bday = datetime(age[0], age[1], age[2])
+            if isinstance(meas_date, (np.ndarray, list)):
+                meas_date = meas_date[0]
+            meas_date = datetime.fromtimestamp(meas_date)
+            subject_age = age_on_date(bday, meas_date)
+        else:
+            subject_age = "n/a"
+
+        data = {'participant_id': [subject_id], 'age': [subject_age],
+                'sex': [sex], 'group': [group]}
+
+        # append the participant data to the existing file if it exists
```

Comment:
I propose dealing with this in the PR on overwriting #43 . Please keep it simple for now and just overwrite by default.